packages/fdc3-agent-proxy/src/listeners/HeartbeatListener.ts

```diff
@@ -26,6 +30,7 @@ export class HeartbeatListener implements RegisterableListener {
         heartbeatEventUuid: (_m as HeartbeatEvent).meta.eventUuid,
       },
     } as HeartbeatAcknowledgementRequest);
+    //console.log("Heartbeat acknowledged")
```

Comment:
Is this supposed to be here? Either delete or add a comment indicating that it was left intentionally, like 
```
// Uncomment when debugging heartbeat messages
```